components/tooling/glean-gradle-plugin/src/main/groovy/mozilla/components/tooling/glean-gradle-plugin/GleanGradlePlugin.groovy

```diff
@@ -175,6 +181,63 @@ subprocess.check_call([
         }
     }
 
+    private Task generateGleanMetricsDocs
+
+    void generateMarkdownDocs(Project project, File condaDir) {
```

Comment:
The Glean sources are generated into the build directory (since they aren't intended to be included in source control), and IIUC, there is a separate build subdirectory for each variant, so we're sort of forced into this.

The markdown docs are in the source tree and intended to be checked into source control.

So, feels to me like there's just an inherent inconsistency we have to live with, but maybe I'm not understanding...

Reply:
It is possible to have multiple variants consume the same generated sources -- it's the equivalent of `src/main` vs `src/{debug,release}`, say.  But it's not the path of least resistance.

Since this really is just a nit, let it ride.